### ISSUE TITLE
please add exit code 

### ISSUE THREAD

**Issue by n0mer** [dundee/mypy2junit#2]
@Dundee Daniel, please return 0 in case there are 0 `mypy` linter messages, and non-zero code otherwize.

W/out proper exit code build pipeline is not failing on CI when there are linter errors.

Thanks.

**Comment by n0mer** [comment]
work-around is to use `pipefail` (https://stackoverflow.com/a/6872163/935546) :
```bash
            - pip install --use-feature=2020-resolver -r requirements.lint.txt
            - mkdir -p ./test-results
            - set -o pipefail
            - python -m mypy --strict . src | mypy2junit > ./test-results/junit.xml
```

**Comment by dundee** [comment]
Good idea. Implemented in 1.5.0 - https://github.com/Dundee/mypy2junit/commit/b4b342dfbf672d58b81b585a9f0003badd29691b.

**Comment by n0mer** [comment]
Thanks!
